pom.xml

```diff
@@ -27,6 +27,11 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
why this complication instead of just "<version>${quarkus-prettytime.version}</version>" on line 56